requirements.txt

```diff
@@ -1,3 +1,3 @@
-python-pptx==0.6.6
```

Comment:
@noboru-fukaishi I think we can remove requirements.txt since we've moved to pyproject.toml :)